spec/controllers/users_controller_spec.rb

```diff
@@ -49,14 +49,14 @@
 
       it "should render the requested user as JSON" do
         expect(User).to receive(:find).and_return(current_user)
-        expect(current_user).to receive(:to_json).and_return("generated JSON")
+        expect_any_instance_of(User).to receive(:to_json).and_return("generated JSON")
```

Comment:
Presumably we don't need 
 expect(User).to receive(:find).and_return(current_user)
if we are no longer stubbing just that record?

Reply:
I'll address this in the final Devise PR.